### ISSUE TITLE
[Feature] Long press to play at 2x speed.

### ISSUE THREAD

**Issue by rohitghali** [moneytoo/Player#568]
Some parts of the video is boring sometimes and still can't be forwarded. Playing at a faster rate would be fantastic in such scenarios. Currently youtube app does this and MX Player also does this really well. Just try it out in MX Player and you'll get an Idea. It'd be fantastic. Maybe a toggle in the settings?

**Comment by githottub** [comment]
Please fully investigate before creating an issue. Playback speed control is already an existing feature.

**Comment by rohitghali** [comment]
> Please fully investigate before creating an issue. Playback speed control is already an existing feature.

Am I not clear enough or you didn't understand a word that I typed? I'm not talking about playback speed control. I'm talking about long tap to speed up playback. They're two different things.

**Comment by githottub** [comment]
No, you are not clear enough. You said a "toggle in the settings" and there already is a toggle in the settings to speed up playback as much as 2x. Which seems to be what you were asking for... a toggle in the settings in offers to play at 2x speed. And literally every single person I have shown this to thinks the same thing... what you are asking for already exists. So, obviously you are not clear enough.

But since you chose to downvote my reply instead of explaining more clearly regarding exactly how what already exists is somehow different than what you want, then it seems like you should do it yourself rather than asking others to help you.

**Comment by rohitghali** [comment]
> No, you are not clear enough. You said a "toggle in the settings" and there already is a toggle in the settings to speed up playback as much as 2x. Which seems to be what you were asking for... a toggle in the settings that offers to play at 2x speed. And literally every single person I have shown this to thinks the same thing... what you are asking for already exists. So, obviously you are not clear enough.
> 
> But since you chose to downvote my reply instead of explaining more clearly regarding exactly how what already exists is somehow different than what you want, then it seems like you should do it yourself rather than asking others to help you.

Dude, I don't know what your problem is. Since your comments are not contributing anything meaningful, I'm gonna ignore you from now on. Let the devs comment. I hope they understand my request.

**Comment by jinggqu** [comment]
Hi @rohitghali, I have forked this repo and implemented 2x speed on long press. If you are interested, you can download from https://github.com/jinggqu/Player/releases. Enjoy.

**Comment by rohitghali** [comment]
> Hi [@rohitghali](https://github.com/rohitghali), I have forked this repo and implemented 2x speed on long press. If you are interested, you can download from https://github.com/jinggqu/Player/releases. Enjoy.

Damn! That works really well. Thanks. I hope this change gets merged into main repo.

**Comment by smed79** [comment]
@jinggqu Thanks for implementing this feature and great to see community contributions like this!

Would you consider submitting a pull request to this repo? It could be a great addition for others who want faster playback, and the maintainer can decide whether it fits with the overall direction of the project.
